urwid/str_util.py

```diff
@@ -35,14 +36,18 @@
 _byte_encoding: Literal["utf8", "narrow", "wide"] = "narrow"
 
 
-def get_width(o: int) -> Literal[0, 1, 2]:
-    """Return the screen column width for unicode ordinal o."""
-    width = wcwidth.wcwidth(chr(o), UNICODE_VERSION)
+def get_char_width(char: str) -> Literal[0, 1, 2]:
+    width = wcwidth.wcwidth(char, UNICODE_VERSION)
     if width < 0:
```

Comment:
control characters width calculated as `-1` (they are not expected in the string for display)